docs/pages/components/Panel.mdx

```diff
@@ -0,0 +1,261 @@
+---
+title: Panel
+description: A common, versatile pattern used to display content composed of different elements, usually in a collection.
+source: https://github.com/dequelabs/cauldron/blob/develop/packages/react/src/components/Panel/index.tsx
+---
+
+import {
+  Panel,
+  PanelHeader,
+  PanelContent,
+  Toast,
+  Button,
+  Icon,
+  Link,
+  IconButton,
+  Code
+} from '@deque/cauldron-react';
+
+```js
+import { Panel, PanelHeader, PanelContent } from '@deque/cauldron-react';
+```
+
+## Examples
+
+### Panel
+
+At its simplest, a `Panel` is a styled semantic HTML `<section>` element with with content inside.
+
+<Note>
+If a `Panel` does not have a heading, <Link href="https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/general-principles.html" rel="noopener noreferrer" target="_blank">it is recommended</Link> to add an `aria-label` to the `Panel` component.
+</Note>
+
+```jsx example
+<Panel aria-label="Panel Heading">
+  <p>Panel Content</p>
+</Panel>
+```
+
+### Panel with Heading (Prop)
+
+To add a heading to a `Panel`, you can use the `heading` prop. One advantage to using this prop over composing
+the elements is it automatically adds an `id` to the heading and `aria-labelledby` to the `Panel` component.
+
+Inspect the below example and you will see that the `Panel` component has an `aria-labelledby` attribute with the 
+value of the `id` of the heading.
+
+```jsx example
+<Panel heading={{ id: 'panel-heading-prop-example', text: 'Panel Heading', level: 3 }}>
+  <p>Panel Content</p>
+</Panel>
+```
+
+### Panel (Composed)
+
+In addition to using the `heading` prop, you can also compose the `Panel` component with the `PanelHeader` and 
+`PanelContent` components.
+
+<Note>
+  The `PanelHeader` and `PanelContent` should not be used with the `Panel`
+  heading prop at the same time. Choose one or the other to use. 
+  
+  In addition, if you use the `PanelHeader`, you are responsible for adding `aria-labelledby` to
+  the `Panel` and an ID to the heading.
```

Comment:
See initial note, we just need to think about this as it's not absolutely required.